### DISCUSSION TITLE
Heading of a Commemoratio

### DISCUSSION THREAD

**Discussion by Michaelskapel** [DivinumOfficium/divinum-officium#4071] [Q&A]
In the Nederlands version, the red-colored heading of a Commemoratio is identical to the English one: **Commemoration** instead of the Latin **Commemoratio** (which is what I prefer); see, for instance, 1 October 2024. Is this another unwanted result of the English detour in our system? How can this be changed?

**Comment by Augustinus-Altovadensis** [comment]
It depends on what are you talking about. If you have the Holy Mass in mind, you can find the word translated in `/DivinumOfficium/divinum-officium/tree/master/web/www/missa/*language*/Ordo/Prayers.txt` By changing that, or rather by introducing following text, you can change it "back":
```
[Commemoratio]
Commemoratio
```

And yes, what you see is a carry-over from English, because the `[Commemoratio]` tag is missing.

**Reply by Michaelskapel** [reply]
@Augustinus-Altovadensis Thank you for your reply!
I followed your advice, but it did only partially solve the problem. Please have a look again to the Nederlands version of 1 October. I now see the word **Commemoratio** in red (as it should be), but the actual text of the commemoratio appears in black (except for the first letter) and on a new line. How can this be solved?

**Reply by FAJ-Munich** [reply]
@Michaelskapel 
I've just checked 1st October Missa on the website and the Commemoration S. Remigii appears to be just fine?

**Reply by Michaelskapel** [reply]
@FAJ-Munich Yes, it is fine now! It seems that it took some time to get everything working correctly. We can now close the discussion.
